android/src/androidTest/java/bbct/android/common/test/rule/DataTestRule.java

```diff
@@ -20,13 +20,16 @@
 
 import android.app.Instrumentation;
 import android.support.test.InstrumentationRegistry;
-import bbct.android.common.test.BaseballCardCsvFileReader;
-import bbct.android.common.test.DatabaseUtil;
-import bbct.data.BaseballCard;
+
+import org.junit.rules.ExternalResource;
+
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.List;
-import org.junit.rules.ExternalResource;
+
+import bbct.android.common.test.BaseballCardCsvFileReader;
+import bbct.android.common.test.DatabaseUtil;
+import bbct.data.BaseballCard;
```

Comment:
I like using Android Studio's default ordering of imports. Continue to organize imports in this way in any files that you edit. Don't worry about doing it for files that you don't change.

Reply:
got it